src/python/director/lcmUtils.py

```diff
@@ -113,7 +113,10 @@ def handleMessage(messageData):
                 # except ValueError:
                 #     pass
         if loadSuccessful:
-            callback(msg)
+            try:
+                callback(msg, channel)
+            except TypeError:
+                callback(msg)
```

Comment:
I think I'd rather handle this by passing an optional argument to addSubscriber that determines whether the callback should be called with one argument or two arguments.

A couple years ago I had a branch where I tried a try/except as above, and determined it had an issue with some callbacks.  I can't remember exactly, but maybe it was for callbacks with optional args, the TypeError check isn't valid.

Another solution could be to require all callbacks to accepts msg, channel args, and refactor all the existing code in director, but I think I'd rather just add an extra optional arg to addSubscriber to handle this less common case where you want the channel passed to the callback.